lib/miq_automation_engine/engine/miq_ae_method.rb

```diff
@@ -225,7 +225,7 @@ def self.ruby_method_runnable?(aem)
 
     def self.setup_drb_for_ruby_method
       require 'drb/timeridconv'
-      DRb.install_id_conv(DRb::TimerIdConv.new(drb_cache_timeout))
+      @@global_id_conv = DRb.install_id_conv(DRb::TimerIdConv.new(drb_cache_timeout))
       drb_front  = MiqAeMethodService::MiqAeServiceFront.new
       drb        = DRb.start_service("druby://127.0.0.1:0", drb_front)
```

Comment:
@mkanoor So, here's a crazy idea... we're currently creating a new TimerIdConv for each automate ruby method call which creates a new thread that lives forever unless we do the crazy thread cleanup later...

Another crazy thing we can do is reuse the same timer thread for each service:
Do we run these methods concurrently?  If not, how about we maintain a single timer object and wake the thread before we attach it to the new service?  Note, if we don't wake it, it could have 10 seconds left in the drb_cache_timeout sleep before it wakes up and makes the objects GC'able.  Maybe something like this:

```ruby
@timer ||= DRb::TimerIdConv.new(drb_cache_timeout)
@timer.instance_variable_get('@holder').instance_variable_get('@keeper').wakeup
drb = DRb.start_service("druby://127.0.0.1:0", drb_front, :idconv => @timer)
```